externals/ssl/src/CVMFS-CMakeLists.txt

```diff
@@ -0,0 +1,5 @@
+include ("${EXTERNALS_LIB_LOCATION}/CMake-Register_External_Lib.txt")
+
+set (SSL_ARCHIVE "${SSL_BUILTIN_LOCATION}/ssl_install/lib/libssl.a;${SSL_BUILTIN_LOCATION}/ssl_install/lib/libcrypto.a")
```

Comment:
I think I covered this in the root `CMakeLists.txt` (line 414). There is a conditional based on platform. On macOS, I set `OPENSSL_LIBRARIES` to the value of `SSL_ARCHIVE`.